test/test.js

```diff
@@ -4,7 +4,7 @@ require("../core_test");
 
 require("../cache-requests/cache-requests_test");
 
-require("../can/test");
+require("../can/test-without-proxy");
```

Comment:
We should run all the tests here, we should only run the `without-proxy` tests in canjs/canjs where we're testing IE11.